src/components/Home/Search/Search.tsx

```diff
@@ -1,7 +1,48 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import { ENDPOINTS } from '../../../helpers/api';
+import { LocationSearch } from '../../../helpers/Interfaces';
+import { useDebounce } from './../../../hooks/useDebounce';
+import { useGetRequest } from './../../../hooks/useGetRequest';
+import SearchResults from './SearchResults/SearchResults';
+import styles from './Search.module.scss';
 
 const Search: React.FunctionComponent = () => {
-  return <input placeholder="Search for location" />;
+  const [searchTerm, setSearchTerm] = useState<string>('');
+  const debouncedSearchTerm: string = useDebounce(searchTerm);
+  const {
+    data: searchResults = { locations: [] },
+    loading = true,
+    error = null
+  }: { data: LocationSearch; loading: boolean; error: string | null } = useGetRequest(
+    ENDPOINTS.locationSearch,
+    debouncedSearchTerm
+  );
+  const [displaySearchResults, setDisplaySearchResults] = useState<boolean>(false);
+
+  const inputChangeHandler = (event: FormEvent<HTMLInputElement>): void => {
+    setDisplaySearchResults(true);
+    setSearchTerm(event.currentTarget.value);
+  };
+
+  return (
+    <section className={styles.container}>
+      <input
+        className={styles.input}
+        placeholder="Search for location"
+        type="text"
+        onChange={inputChangeHandler}
+        value={searchTerm}
+      />
+      {searchTerm.length < 3 && <p>Enter at least 3 letters</p>}
+      {searchResults.locations.length > 0 && displaySearchResults && (
```

Comment:
`searchResults.locations.length > 0` can be simplified

Reply:
`!!searchResults.locations.length`
By the way, I was thinking to add a loading state here also, so maybe will change this later.